### PR TITLE
Fixing FX rate decimals to 4 for display

### PR DESCRIPTION
**Pull request by aparthibanpaypal**

### CODE REVIEW

File: Common/Sources/Extensions/String.swift [hunk 1]
```diff
@@ -80,6 +80,19 @@ public extension String {
...
         return Double(result) ?? 0
     }
+
+    /// Get FX Rate with four decimal
+    /// - Returns: return string with four decimal
+    func getFxRateWithFourDecimal() -> String {
+        guard !self.isEmpty else {
+            return ""
+        }
+        let decimals = self.split(separator: ".")[1]
```
[the remark this answers is not in the document]
Comment (aparthibanpaypal) [reply Common/Sources/Extensions/String.swift]: Fixed it.

File: UITests/TransferFunds/TransferUserFundsConfirmationTest.swift [hunk 2]
```diff
@@ -138,28 +138,28 @@ class TransferUserFundsConfirmationTest: BaseTests {
         XCTAssertTrue(app.cells.element(boundBy: 3).staticTexts[youBuy].exists)
         XCTAssertTrue(app.cells.element(boundBy: 3).staticTexts["$2,337.93 USD"].exists)
         XCTAssertTrue(app.cells.element(boundBy: 4).staticTexts[exchangeRate].exists)
-        XCTAssertTrue(app.cells.element(boundBy: 4).staticTexts["\u{200E}RM1 MYR = $0.233968 USD"].exists)
+        XCTAssertTrue(app.cells.element(boundBy: 4).staticTexts["\u{200E}RM1 MYR = $0.2339 USD"].exists)
```
[the remark this answers is not in the document]
Comment (aparthibanpaypal) [reply UITests/TransferFunds/TransferUserFundsConfirmationTest.swift:141]: Tested in local. It is working as expected. 
Screenshot 2021-03-15 at 11 43 34 AM

**Review by rbanikPP — APPROVED** [review]

**Comment by ddhar-hw** [comment]
changes look good to me. Kindly make sure that this is working for locales that use dot in place of comma

File: Demo/IntegratorAuthenticationProvider.swift [hunk 3]
```diff
@@ -21,8 +21,8 @@ import HyperwalletSDK
 
 public class IntegratorAuthenticationProvider: HyperwalletAuthenticationTokenProvider {
     private var url: String
-    private let user: String = "userName"
-    private let password: String = "password"
+    private let user: String = "selrestuser@54687411683"
+    private let password: String = "Password1!"
     private let session: URLSession
 
     init(_ baseUrl: String, _ userToken: String) {
```
Comment (malaw-hw) [inline Demo/IntegratorAuthenticationProvider.swift]: I think we are not supposed to update this

**Review by malaw-hw — CHANGES_REQUESTED** [review]

File: Demo/IntegratorAuthenticationProvider.swift (hunk 3 above)
Comment (aparthibanpaypal) [reply Demo/IntegratorAuthenticationProvider.swift]: Will check and revert it. Thanks.

**Review by SaiNarendran — APPROVED** [review]

**Review by malaw-hw — APPROVED** [review]

**Review by ddhar-hw — APPROVED** [review]

**Pull request merged**